[2025-07-26T10:00:11.801+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-26T10:00:11.822+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_postgres_test2.load_songplays_fact_table scheduled__2025-07-26T09:00:00+00:00 [queued]>
[2025-07-26T10:00:11.835+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_postgres_test2.load_songplays_fact_table scheduled__2025-07-26T09:00:00+00:00 [queued]>
[2025-07-26T10:00:11.836+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-07-26T10:00:11.855+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_songplays_fact_table> on 2025-07-26 09:00:00+00:00
[2025-07-26T10:00:11.866+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1204) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-26T10:00:11.868+0000] {standard_task_runner.py:72} INFO - Started process 1205 to run task
[2025-07-26T10:00:11.866+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_postgres_test2', 'load_songplays_fact_table', 'scheduled__2025-07-26T09:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/my_dags.py', '--cfg-path', '/tmp/tmp0cjal1r4']
[2025-07-26T10:00:11.871+0000] {standard_task_runner.py:105} INFO - Job 54: Subtask load_songplays_fact_table
[2025-07-26T10:00:11.966+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_postgres_test2.load_songplays_fact_table scheduled__2025-07-26T09:00:00+00:00 [running]> on host 2f3d4b2b9190
[2025-07-26T10:00:12.144+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_postgres_test2' AIRFLOW_CTX_TASK_ID='load_songplays_fact_table' AIRFLOW_CTX_EXECUTION_DATE='2025-07-26T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-26T09:00:00+00:00'
[2025-07-26T10:00:12.145+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-26T10:00:12.175+0000] {my_dags.py:386} INFO - Loading songplays fact table
[2025-07-26T10:00:12.358+0000] {my_dags.py:455} INFO - Inserted 6820 records into songplays table
[2025-07-26T10:00:12.361+0000] {my_dags.py:459} INFO - Records with matched songs: 1
[2025-07-26T10:00:12.370+0000] {my_dags.py:462} INFO - Songplays fact table loaded successfully
[2025-07-26T10:00:12.371+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-26T10:00:12.387+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-26T10:00:12.389+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_postgres_test2, task_id=load_songplays_fact_table, run_id=scheduled__2025-07-26T09:00:00+00:00, execution_date=20250726T090000, start_date=20250726T100011, end_date=20250726T100012
[2025-07-26T10:00:12.447+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-26T10:00:12.482+0000] {taskinstance.py:3895} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-07-26T10:00:12.485+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
